workflow-testing/src/main/java/com/squareup/workflow1/testing/RenderTester.kt

```diff
@@ -262,6 +267,11 @@ public abstract class RenderTester<PropsT, StateT, OutputT, RenderingT> {
     block: (rendering: RenderingT) -> Unit = {}
   ): RenderTestResult<PropsT, StateT, OutputT>
 
+  public open fun requireExplicitWorkerExpectations():
```

Comment:
This should be abstract, like most of the methods in this class.

Reply:
Like above, wasn't clear on intention.